### ISSUE TITLE
Currency list has some odd entries

### ISSUE THREAD

**Issue by j2tausig** [mitodl/micromasters#1567]
The currency list that is displayed in the Cost Calculator has some odd entries:
*[8]	Malagasy ariary
*[8]	Mauritanian ouguiya
Gold (one troy ounce)
Palladium (one troy ounce)
Platinum (one troy ounce)
Silver (one troy ounce

Steps to Reproduce:
Log into MicroMasters
Fill out a profile and select a program.
On the Dashboard page, click on "Calculate Cost" and view the Currency Drop down.

**Comment by pdpinch** [comment]
@aliceriot can we just remove these from the front-end?

**Comment by aliceriot** [comment]
Yeah, there's a list of currency codes to exclude in `static/js/lib/currency.js`.
